src/ActionRow/README.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'ActionRow'
+type: 'component'
+components:
+- ActionRow
+- ActionRowSpacer
+categories:
+- Buttonlike
+status: 'Stable'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+
+---
+
+A layout utility for the common use case of aligning buttons, links or text
+in a row in a control bar or nav.
+### Basic Usage
+
+```jsx live
+<ActionRow>
+  <Form.Checkbox>Don't ask me again.</Form.Checkbox>
+  <ActionRow.Spacer />
+  <Button variant="tertiary">
+    Cancel
+  </Button>
+  <Button variant="primary">
+    Submit
+  </Button>
+</ActionRow>
+```
+
+### Stacked Usage
+
+
+```jsx live
+<ActionRow isStacked>
+  <p className="x-small">
+    Bespoke leggings yuccie, portland umami readymade craft beer vaporware sriracha.
+  </p>
+  <Button variant="tertiary">
+    Go back
```

Comment:
On the netlify preview, this button is smaller in size than the continue button, which irks my soul. Also it's below it, which I wouldn't expect from the code, so I would either put them in the other order or make a note of that in the docs.

Reply:
![Screen Shot 2021-04-12 at 1 55 19 PM](https://user-images.githubusercontent.com/13922520/114439199-bccec800-9b96-11eb-9e79-6868809326a1.png)

(In the ActionRow docs)

Reply:
I can note that in the docs! We aim to put the last item as the "primary" action. Horizontally that means the end in our language and vertically it means the top of the stack.

As far as size, these buttons should not be the same width. They happen to be close which is unfortunate, but right now our styles are not to assign arbitrary widths to buttons (either full width or natural width)